charts/aws-efs-csi-driver/templates/controller-deployment.yaml

```diff
@@ -13,6 +13,10 @@ spec:
       app: efs-csi-controller
       app.kubernetes.io/name: {{ include "aws-efs-csi-driver.name" . }}
       app.kubernetes.io/instance: {{ .Release.Name }}
+  {{- with .Values.controller.updateStrategy }}
+  strategy:
+    {{ toYaml . | nindent 4 }}
```

Comment:
are we missing a dash `-` before `toYaml` here? same as below

Reply:
No, this looks good.  `-` kills white space but I think we want that here